addons/spark/0.x/spark.yaml

```
@@ -0,0 +1,31 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: spark
+  labels:
+    kubeaddons.mesosphere.io/name: spark
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    stage.kubeaddons.mesosphere.io/spark: Beta
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.1.0-2"
```

Revision should be the app version with the revision number after, I believe this would be the appropriate revision?
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "0.2.0-1"
```